rtv/subreddit.py

```diff
@@ -73,6 +74,24 @@ def prompt_subreddit(self):
         if name is not None:
             self.refresh_content(order='ignore', name=name)
 
+    @SubredditController.register(Command('SUBREDDIT_FRONTPAGE'))
+    def show_frontpage(self):
```

Comment:
Ahh I think I see what the problem is. $PYTHONPATH is just supposed to point to the current directory, it shouldn't be that whole string. The reasoning is that you want your test files to search in your current directory first when they do ``import rtv``, instead of loading it from your site packages or wherever you installed rtv with pip.

```bash
$ export PYTHONPATH=.
$ echo $PYTHONPATH
.
$ py.test --record-mode=once --refresh-token=~/refresh-token
================================ test session starts ================================
platform linux -- Python 3.4.0, pytest-2.9.1, py-1.4.31, pluggy-0.3.1
rootdir: /home/michael/Projects/rtv_project, inifile: 
plugins: xdist-1.14, cov-2.2.0
collected 113 items 

tests/test_config.py ......
```